SofaKernel/framework/sofa/core/objectmodel/BaseClass.cpp

```diff
@@ -22,6 +22,10 @@
 #include <sofa/core/objectmodel/BaseClass.h>
 #include <sofa/helper/logging/Messaging.h>
 
+#ifdef __GNUC__
```

Comment:
Why ?

Reply:
This is a compiling fix for Mac OSX.
This comes from a merge of PR #969  in this branch.